src/commander/flags.cr

```diff
@@ -11,7 +11,15 @@ class Commander::Flags
     flags.each { |flag| yield flag }
   end
 
-  def add(flag)
+  def add(*multi)
+    add multi.to_a
+  end
+
+  def add(multi : Array(Commander::Flag))
```

Comment:
The only reason I didn't here is that it "clashes" with the getter. I think I originally used the getter then refactored. I'll update.